### PR TITLE
Initial release

### PR DESCRIPTION
**Pull request by pbugni**
docker-compose based instance of [PosgtREST](http://postgrest.org/en/latest/), built from https://github.com/suzel/docker-postgrest

exposes an `event` API intended to provide easy access to POST / GET JSON logging events.

### CODE REVIEW

File: config/postgrest.conf [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# postgrest.conf
```
Comment (ivan-c) [inline config/postgrest.conf:1]: I don't think this file is doing anything (without a bind mount)?

File: docker-compose.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    environment:
+      PGRST_DB_URI: postgres://app_user:secret@postgres:5432/app_db
+      PGRST_DB_SCHEMA: api
+      PGRST_DB_ANON_ROLE: web_anon
+      PGRST_JWT_SECRET: ${PGRST_JWT_SECRET}
+    links:
+      - postgres:postgres
+
+  postgres:
+    image: postgres:12.3
```
Comment (ivan-c) [inline docker-compose.yaml:18]: Would stick with `12` for automatic minor version updates

File: config/postgrest.conf (hunk 1 above)
Comment (pbugni) [reply config/postgrest.conf]: removed.

File: docker-compose.yaml (hunk 2 above)
Comment (pbugni) [reply docker-compose.yaml]: ✔️

**Pull request merged**